test/groovy/com/sap/piper/integration/TransportManagementServiceTest.groovy

```diff
@@ -31,43 +31,50 @@ class TransportManagementServiceTest extends BasePiperTest {
 
     @Test
     void retrieveOAuthToken__successfully() {
+        def uaaUrl = 'http://dummy.com/oauth'
+        def clientId = 'myId'
+        def clientSecret = 'mySecret'
+        def expectedToken = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIn0.Gfx6VO9tcxwk6xqx9yYzSfebfeakZp5JYIgP_edcw_A'
+        def responseContent = '{ "access_token": "' + expectedToken + '" }'
+
         Map requestParams
         helper.registerAllowedMethod('httpRequest', [Map.class], { m ->
             requestParams = m
-            return [content: '{ "access_token": "myOAuthToken" }', status: 200]
+            return [content: responseContent, status: 200]
         })
 
-        def uaaUrl = 'http://dummy.com/oauth'
-        def clientId = 'myId'
-        def clientSecret = 'mySecret'
-
-        def tms = new TransportManagementService(nullScript, [:])
+        def tms = new TransportManagementService(nullScript, [verbose: false])
         def token = tms.authentication(uaaUrl, clientId, clientSecret)
 
         assertThat(loggingRule.log, containsString("[TransportManagementService] OAuth Token retrieval started."))
         assertThat(loggingRule.log, containsString("[TransportManagementService] OAuth Token retrieved successfully."))
-        assertThat(token, is('myOAuthToken'))
+        assertThat(loggingRule.log, not(containsString(expectedToken)))
```

Comment:
Basically the same applies here. Adding this line below

```
assertThat(loggingRule.log, not(containsString('myOAuthToken')))
```

does the job.

Reply:
Addressed in the 2nd test as well.